java/cdk/src/main/java/sleeper/cdk/stack/bulkimport/CommonEmrBulkImportHelper.java

```diff
@@ -129,6 +133,10 @@ public IFunction createJobStarterFunction(String bulkImportPlatform, Queue jobQu
         importBucket.grantReadWrite(function);
         addIngestSourceBucketReferences(scope, "IngestBucket", instanceProperties)
                 .forEach(ingestBucket -> ingestBucket.grantRead(function));
+        statusStore.grantWriteJobEvent(commonEmrStack.getEmrRole());
+        statusStore.grantWriteJobEvent(commonEmrStack.getEc2Role());
+        statusStore.grantWriteTaskEvent(commonEmrStack.getEmrRole());
+        statusStore.grantWriteTaskEvent(commonEmrStack.getEc2Role());
```

Comment:
It seems like the grants for the common bulk import stack can be managed there. It doesn't look like the status store resources are needed in this class.